Sources/App/app.swift

```diff
@@ -9,3 +9,20 @@ public func app(_ env: Environment) throws -> Application {
     let app = try Application(config: config, environment: env, services: services)
     return app
 }
+
+extension Environment {
+    enum Key: String {
+        case SLACK_TOKEN
+        case SLACK_CHANNEL
+        case GITHUB_USERNAME
+        case GITHUB_TOKEN
+        case CIRCLECI_TOKEN
+        case JIRA_HOST
```

Comment:
If it's not longer just the host name that we are expecting (e.g. `foo.atlassian.net`) but the `baseURL` now (e.g. `https://foo.atlassian.net:443/`), as your changes in this PR seems to suggest, then the name of the env variable should reflect that change too

Reply:
it's the same

Reply:
How would you suggest to rename it?

Reply:
If we indeed go with the change of it not just being expect to contain just the host name but the baseURL, then what about renaming the env var `JIRA_BASEURL` then?

Reply:
```suggestion
        case JIRA_BASEURL
```

Reply:
done